recipes/expansionhunter/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+boost:
+  - 1.82
```

Comment:
Is this strict version really needed? It will restrict condas resolver.

Reply:
So, there's probably some flexibility. The package by default pulls in 1.73 source and builds it internally; clearly this is not ideal for various reasons. Some of the patches are required due to boost version changes. Boost is great but obviously loves to introduce substantial minor version breaking changes. I'm somewhat ambivalent. If we remove boost pinning entirely, if the package has to be rebuilt at some point, boost versions far enough in the future will almost certainly fail without further patching. 1.83 might be fine, but I'd be surprised if 1.90 ends up being compatible as-is. I'm open to guidance here, if you have preferences. Unpinning entirely and just dealing with it in the future is an option.